index.html

```diff
@@ -21,8 +21,8 @@ <h1 id="score">Score = 0</h1>
     <div class="col-md-8 gameboard">
       <center>
         <h1 class="cover-heading">My Ping Pong Game</h1>
-        <!-- <canvas id="pong" data-processing-sources="testgame.pde"/></canvas> -->
-        <canvas id="pong" data-processing-sources="pong.pde"/></canvas>
+        <canvas id="pong" data-processing-sources="testgame.pde"/></canvas>
```

Comment:
This one line change doesn't really need a review or a comment, but I want to build a good habit of always doing code review before moving forward with a merge request.